### ISSUE TITLE
Color area is still selectable after closing the picker

### ISSUE THREAD

**Issue by makepanic** [Simonwep/pickr#126]
#### Do you want to request a **feature** or report a **bug**? 

* bug

#### What is the current behavior?

After dragging the color selector in the color area and closing the picker, it seems like the listener still exists and the user is able to change the selector further.

![pickr](https://user-images.githubusercontent.com/1205444/61631168-ec82c180-ac89-11e9-8065-27a08ac15f92.gif)

#### If the current behavior is a bug, please provide the steps to reproduce and 

1. open the picker
2. drag the color selector around
3. close the picker
4. move your mouse over the previously visible color selector area
5. move the mouse around while dragging

#### What is the expected behavior?

It shouldn't be possible to drag the color selector after closing the picker.
I think this is also an issue because the content below the color area isn't usable after closing it.

#### Your environment:
```
Version (see Pickr.version): 1.2.3
Used bundle (es5 or normal one): es5
Used theme (default is classic): classic
Browser-version:  chromium 75
Operating-system:  linux
```

**Comment by Simonwep** [comment]
I'd almost say this is a chrome bug. If you take a look at the dev console, you see pickr has `visibility: hidden` which should deny any interaction. If you unfocused the tab this isn't possible anymore. And check it out in firefox and edge, the issue seems to be only with chrome...

**Comment by makepanic** [comment]
Just checked it in FF and it also happens there :(
There's no cursor adjustment though.

![pickr-ff](https://user-images.githubusercontent.com/1205444/61632150-70d64400-ac8c-11e9-8707-c84538426b25.gif)

**Comment by Simonwep** [comment]
Oh, my bad. Seems like I forgot to adjust the delay of `visibility` - and I thought I'd finally discover a chrome bug 😅 Fixed in e1c0eeb39b352c3d0511780d7b194d8de8f638f4
